addons/mass_mailing/models/mailing.py

```diff
@@ -667,3 +731,73 @@ def _image_to_url(b64image: bytes):
             return lxml.html.tostring(root)
 
         return body_html
+
+    def _prepare_statistics_email_values(self):
+        """Return some statistics that will be displayed in the mailing statistics email.
+
+        Each item in the returned list will be displayed as a table, with a title and
+        1, 2 or 3 columns.
+        """
+        self.ensure_one()
+
+        random_tip = self.env['digest.tip'].search(
+            [('group_id.category_id', '=', self.env.ref('base.module_category_marketing_email_marketing').id)]
+        )
+        if random_tip:
+            random_tip = random.choice(random_tip).tip_description
+
+        formated_date = tools.format_datetime(
+            self.env, self.sent_date, self.user_id.tz, 'MMM dd, YYYY',  self.user_id.lang
+        ) if self.sent_date else False
+
+        engagement_title = self.env['ir.qweb']._render(
+            "mass_mailing.mass_mailing_kpi_title",
+            {"title": _('Engagement'), "sent": self.sent}
+        )
+
+        business_benefits_title = self.env['ir.qweb']._render(
+            "mass_mailing.mass_mailing_kpi_title",
+            {"title": _('Business Benefits'), "sent": self.sent}
+        )
+
+        top_button = self.env['ir.qweb']._render(
+            'mass_mailing.mass_mailing_more_info',
+            {
+                'web_base_url': self.env['ir.config_parameter'].sudo().get_param('web.base.url'),
+                'mailing_id': self.id,
+            },
+        )
+
+        return {
+            'title': _('24H Stats of mailing'),
+            'sub_title': '"%s"' % self.subject,
+            'top_button': top_button,
+            'kpi_data': [
+                {
+                    'kpi_fullname': engagement_title,
+                    'kpi_fullname_raw': True,
+                    'kpi_action': None,
+                    'kpi_col1': {
+                        'value': f'{self.received_ratio}%',
+                        'col_subtitle': '%s (%i)' % (_('RECEIVED'), self.delivered),
+                    },
+                    'kpi_col2': {
+                        'value': f'{self.opened_ratio}%',
+                        'col_subtitle': '%s (%i)' % (_('OPENED'), self.opened),
+                    },
+                    'kpi_col3': {
+                        'value': f'{self.replied_ratio}%',
+                        'col_subtitle': '%s (%i)' % (_('REPLIED'), self.replied),
+                    },
+                }, {
+                    'kpi_fullname': business_benefits_title,
+                    'kpi_fullname_raw': True,
+                    'kpi_action': None,
+                    'kpi_col1': {},
+                    'kpi_col2': {},
+                    'kpi_col3': {},
+                },
+            ],
+            'tips': [random_tip],
+            'formated_date': formated_date,
```

Comment:
I think it is labelled formatTed, with 2 "T" (to propagate to all other formated)